downloads/verification-notes/release-artfiacts-verification.md

```diff
@@ -0,0 +1,79 @@
+# Official Ballerina {{ version }} Release Artifacts
```

Comment:
- The version is set as placeholder, with the metadata.json it'll auto update the template with the relevant version. so we can leave it as it is.
- Initially my suggestion was to only have `Signatures and Verification` section displayed on the website or it would be repetitive, whilst including the `certificate` and `signature` files within the download page where previously the `pgp` keys were presented. However this was planned as an improvement.

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-dev-website/pull/7256.